doc/sphinx/addendum/type-classes.rst

```diff
@@ -336,30 +336,29 @@ Summary of the commands
 
    .. warn:: A coercion will be introduced instead of an instance in future versions when using ':>' in 'Class' declarations. Replace ':>' with '::' (or use '#[global] Existing Instance field.' for compatibility with Coq < 8.17).
 
-      In future versions, :g:`:>` will
-      declare a :ref:`coercion<coercions>`, as it does
-      for other :cmd:`Record` commands.
-      To eliminate the warning,
-      use :g:`::`.
+      In future versions, :g:`:>` will declare a :ref:`coercion<coercions>`, as
+      it does for other :cmd:`Record` commands. To eliminate the warning, use
+      :g:`::` instead.
 
    .. warn:: Ignored instance declaration for “@ident”: “@term” is not a class
 
-      Using this ``::`` (or deprecated ``:>``) syntax with a right-hand-side that is not itself a Class
-      has no effect (apart from emitting this warning).
+      Using the ``::`` (or deprecated ``:>``) syntax with a right-hand-side that
+      is not itself a Class has no effect (apart from emitting this warning).
 
 .. cmd:: Instance {? @ident_decl {* @binder } } : @type {? @hint_info } {? {| := %{ {* @field_val } %} | := @term } }
 
-   Declares a typeclass instance named
-   :token:`ident_decl` of the class :n:`@type` with the specified parameters and with
-   fields defined by :token:`field_val`, where each field must be a declared field of
-   the class.
+   Declares a typeclass instance named :token:`ident_decl` of the typeclass
+   :n:`@type` with the specified parameters and with fields defined by
+   :token:`field_val`, where each field must be a declared field of the
+   typeclass.
 
-   Adds one or more :token:`binder`\s to declare a parameterized instance. :token:`hint_info`
-   may be used to specify the hint priority, where 0 is the highest priority as for
-   :tacn:`auto` hints. If the priority is not specified, the default is the number
-   of non-dependent binders of the instance.  If :token:`one_pattern` is given, terms
-   matching that pattern will trigger use of the instance.  Otherwise,
-   use is triggered based on the conclusion of the type.
+   Adds one or more :token:`binder`\s to declare a parameterized instance.
+   :token:`hint_info` may be used to specify the hint priority, where 0 is the
+   highest priority as for :tacn:`auto` hints. If the priority is not specified,
```

Comment:
```suggestion
   :token:`hint_info` may be used to specify the hint priority. If the priority is not specified,
```

The wording that I removed should appear where `hint_info` is defined.  Doesn't seem necessary to say that here.

Reply:
IMO If it's specific to this command, it goes here.  If it's a general fact about hints, it should be in the general hints documentation.  Repeating it here doesn't seem necessary.  A few things are worth repeating, but stating the same facts in multiple places can easily lead to them becoming inconsistent.

Reply:
OK seems that there is no documentation (that I could find) about hint priorities. I am inclined to leave this here, and make a note of the change for another PR.